rosbridge_server/scripts/rosbridge_websocket_client.py

```diff
@@ -91,20 +97,28 @@ def send_message(self, _message):
 
 class VideoTransfer():
     def __init__(self, url, args, connection):
-        self.conn = connection
-        url = url + "?" + urllib.urlencode(args)
-        url = url.replace("%2F","/")
-        req = tornado.httpclient.HTTPRequest(
-            url = url,
-            streaming_callback = self.streaming_callback)
-        http_client = tornado.httpclient.AsyncHTTPClient()
-        http_client.fetch(req, self.async_callback)
-        print "fetch finished"
+        try:
+            tornado.httpclient.AsyncHTTPClient.configure(
+                    "tornado.curl_httpclient.CurlAsyncHTTPClient")
+
+            self.conn = connection
+            url = url + "?" + urllib.urlencode(args)
+            url = url.replace("%2F","/")
+            req = tornado.httpclient.HTTPRequest(
+                url = url,
+                streaming_callback = self.streaming_callback,
+                connect_timeout = 0.0,
+                request_timeout = 0.0)
+            http_client = tornado.httpclient.AsyncHTTPClient()
+            http_client.fetch(req, self.async_callback)
+            self.start = time.time()
+        except Exception as e:
```

Comment:
@pablisho please raise the exception and handle properly instead of just printing out - i.e.: what happens here with the caller of this constructor when there is an exception?